Python/ogusa/macro_output.py

```diff
@@ -55,9 +55,9 @@ def dump_diff_output(baseline_dir, policy_dir):
     
     ## Load SS results
     ss_policy_path = os.path.join(policy_dir, "SSinit", "ss_init_vars.pkl")
-    ss_policy = pickle.load(open( tpi_macro_vars_policy_path, "rb" ))
+    ss_policy = pickle.load(open( ss_policy_path, "rb" ))
```

Comment:
That's right @talumbau, before the change that wrong file was loaded, which would cause an error in calculating the SS values.